tests/services/label.service.spec.ts

```diff
@@ -150,6 +152,38 @@ describe('LabelService: getColorOfLabel()', () => {
   });
 });
 
+describe('LabelService: getLabelDefinition()', () => {
+  beforeEach(() => {
+    labelService = new LabelService(null);
+  });
+
+  afterEach(() => {
+    labelService = null;
+  });
+
+  it('should return the correct label definition for type.FunctionalityBug', () => {
+    expect(labelService.getLabelDefinition(LabelConstant.TYPE_FUNCTIONALITY_BUG, LabelConstant.TYPE)).toEqual(
+      LabelConstant.DEFINITION_FUNCTIONALITY_BUG
+    );
+  });
+
+  it('should return the correct label definition for severity.Medium', () => {
+    expect(labelService.getLabelDefinition(LabelConstant.SEVERITY_MEDIUM, LabelConstant.SEVERITY)).toEqual(
+      LabelConstant.DEFINITION_SEVERITY_MEDIUM
+    );
+  });
+
+  it('should return a null for label with no definition', () => {
+    expect(labelService.getLabelDefinition(LabelConstant.STATUS_DONE, LabelConstant.STATUS)).toEqual(LabelConstant.DEFINITION_UNDEFINIED);
+  });
+
+  it('should return a null string for invalid inputs', () => {
```

Comment:
```suggestion
  it('should return null for invalid inputs', () => {
```

Reply:
Thank you for pointing this out! My bad for forgetting to remove the string part :(